Open Judge System/Web/OJS.Web/Areas/Contests/Controllers/ResultsController.cs

```diff
@@ -372,7 +373,7 @@ public ActionResult Stats(int contestId, bool official)
 
                 foreach (var problems in contestResults.Problems.GroupBy(p => p.ProblemGroupId))
                 {
-                    var maxResultsForProblemGroupCount = contestResults.Results
+                    var maxResultsForProblemGroupCount = contestResults.Results.ToList()
```

Comment:
For `Count` we don't have to `ToList` the IQueryable, just refactor the `.?` check.